learning_paths/admin.py

```diff
@@ -45,10 +56,49 @@ class RequiredSkillInline(admin.TabularInline):
     model = RequiredSkill
 
 
+class EnrolledUsersInline(admin.TabularInline):
```

Comment:
@mavidser I think it might make sense to not have enrolled users as an inline, but as a separate admin, for two reasons:

1. Large number of enrolled users would clutter and slow down the page
2. Accidentally enrolling a user while editing the Learning Path should not be as easy as selecting something in the dropdown

Reply:
Makes sense. I've added a separate Admin form for user enrollment. Search fields can be used to get the enrollments for a particular learning path or user